### PR TITLE
Update simulations documentation after removing the JVM options settings

### PR DESCRIPTION
**Pull request by guilgaly**
ref: MISC-382

Motivation:

- Users can no longer set JVM options.
- Wording referring to the injector properties has changed in places to be more consistent

Modifications:

- Remove mentions of JVM settings from the documentation
- Update screenshots and references to the injector properties

### CODE REVIEW

File: content/reference/user/simulations/index.md [hunk 1]
```diff
@@ -94,12 +94,11 @@ You can add several pools with different numbers of injectors to run your simula
 
 After this step, you can save the simulation, or click on *Next* to access optional configurations.
 
-### Step 3 & 4: JVM options & Injector Parameters
+### Step 3: Injector Parameters
 
-These steps allow you to define the JVM options, Java system properties and environment variables used when running this particular simulation. You can choose to override the global properties.
+This step allows you to define the Java system properties and environment variables used when running this particular simulation. Properties/variables entered here will add to the defaults, unless you choose to ignore the defaults. If you keep the defaults, and you add a property/variable with the same key as one from the defaults, the simulation's value will be used (it overrides the default).
```
Comment (guilgaly) [inline content/reference/user/simulations/index.md:99]: Tried to clarify what happens here, not sure if it's better?

**Comment by bastien-gatling** [comment]
I don't know if it matters, but concerning [content/reference/user/simulations/images/create-simulation-general.png](https://github.com/gatling/frontline-cloud-doc/pull/114/files?short_path=cc1a131#diff-cc1a1316e6933d11d9cbb3f94674658babdb7a05a06fde43628cfbcc29b7b547), the capture is from a simulation edition whereas it was that of a creation, which has small differences (the save button, for example).

**Review by franciscotco — APPROVED** [review]

File: content/reference/user/simulations/index.md (hunk 1 above)
Comment (bastien-gatling) [reply content/reference/user/simulations/index.md:99]: Let's see if i understood well ;) :
 
Give me a try: 

Properties and variables entered here will be added to this particular simulation.
If you previously ask to keep the defaults properties, they will be added to the default one and you have the possibility to override them here.

Comment (Isammoc) [reply content/reference/user/simulations/index.md:99]: Erf, we do have an UX issue, here.
The same toggle affects two (three?) different set of values.
Perhaps should we add more toggles or merge the different views?

Comment (slandelle) [reply content/reference/user/simulations/index.md:99]: I'd rather ship this and have users' feedback than spending more time to have something perfect and unused.

**Review by Isammoc — APPROVED** [review]

**Comment by guilgaly** [comment]
> I don't know if it matters, but concerning content/reference/user/simulations/images/create-simulation-general.png, the capture is from a simulation edition whereas it was that of a creation, which has small differences (the save button, for example).

Also, the form's title of the changes. I'll redo these screenshot tomorrow from home (because I won't get the same screenshots on the Mac's internal retina screen as on a non-retina external screen...)

**Review by bastien-gatling — APPROVED** [review]

**Review by tpetillot — APPROVED** [review]

**Pull request merged**